pkgs/data/fonts/nasin-nanpa/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib, stdenvNoCC, fetchurl }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "nasin-nanpa";
+  version = "2.5.1";
+
+  srcs = [
+    (fetchurl {
+      name = "${pname}.otf";
+      url = "https://github.com/ETBCOR/${pname}/releases/download/n${version}/${pname}-${version}.otf";
+      hash = "sha256-++uOrqFzQ6CB/OPEmBivpjMfAtFk3PSsCNpFBjOtGEg=";
+    })
+    (fetchurl {
+      name = "${pname}-lasina-kin.otf";
+      url = "https://github.com/ETBCOR/${pname}/releases/download/n${version}/${pname}-${version}-lasina-kin.otf";
+      hash = "sha256-4WIX74y2O4NaKi/JQrgTbOxlKDQKJ/F9wkQuoOdWuTI=";
+    })
+  ];
+
+  dontUnpack = true;
+
+  installPhase = ''
+    mkdir -p $out/share/fonts/opentype
+    for f in $srcs; do
+        n=$(stripHash $f)
+        cp $f $out/share/fonts/opentype/$n
+    done
+  '';
+
+  outputHashAlgo = "sha256";
+  outputHashMode = "recursive";
+
+  meta = with lib; {
+    homepage = "https://github.com/ETBCOR/${pname}";
+    description =
+      "UCSUR OpenType monospaced font for the Toki Pona writing system, Sitelen Pona";
+    longDescription = ''
+      ni li nasin pi sitelen pona.
+      sitelen ale pi nasin ni li sama mute weka.
+      sitelen pi nasin ni li lon nasin UCSUR kin.
```

Comment:
This needs an english translation

Reply:
It is a translation of the short description to toki pona.

Back-translating, it reads
> This is a font/style/system for writing sitelen pona.
> All the characters in this font are the same amount of space. [which is to say, they're monospaced]
> The symbols of this font are available via the [UCSUR](https://www.kreativekorp.com/ucsur/) system, too.